pmg/static/resources/javascript/pmg.js

```diff
@@ -12,6 +12,48 @@ $(function() {
 		});
 	});
 
+
+	var csrftoken = $('meta[name=csrf-token]').attr('content')
+
+	$.ajaxSetup({
+	    beforeSend: function(xhr, settings) {
+	        if (!/^(GET|HEAD|OPTIONS|TRACE)$/i.test(settings.type) && !this.crossDomain) {
+	            xhr.setRequestHeader("X-CSRFToken", csrftoken)
+	        }
+	    }
+	})
+
+	$(".create-alert").on("click", function(e) {
+
+		var q = $(this).data('q'),
+			  committee_id = parseInt($(this).data('committee')) || "",
```

Comment:
You don't need to worry about parsing ints, all form data gets sent as a string.